tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2Tests.cs

```diff
@@ -56,6 +58,9 @@ public async Task SubmitTraces(string packageVersion)
                         // Check the span count
                         Assert.Equal(expectedSpanCount, spans.Count);
 
+                        var settings = VerifyHelper.GetCIVisibilitySpanVerifierSettings(packageVersion);
```

Comment:
What does this have to do with running tests in parallel?

Reply:
So because of the errors I got for `Datadog.Trace.ClrProfiler.IntegrationTests.CI.MsTestV2Tests` and the PR I found from Tony working on the tests to change to snapshots(https://github.com/DataDog/dd-trace-dotnet/pull/5226) I tried merging the branch to mine to see if the tests didn't error out, but they still did... luckily Tony just replied on Slack that he is aware of the errors and looking into those.

Reply:
I did a rebase of my PR, can you do another take to check if that solves the issue?

Reply:
forget it, I need to rework on the snapshots... will do that tomorrow.

Reply:
Dale yo confio.

Reply:
My PR with the fixes should be passing now, you can test that branch to check if the issue is solved.